### PR TITLE
implemented bla bla

### PR DESCRIPTION
**Pull request by bktansuki**
Finished implementing Logic

### CODE REVIEW

**Comment by chinloong93** [comment]
BK Good Job. Approved.

File: GuessingGame.cpp [hunk 1]
```diff
@@ -12,7 +12,14 @@ int GenerateRandomNumber() {
 // If guess is smaller than randomNumber, print to console "Too low."
 // If guess us equal to randomNumber, print to console "That's right"
 void GameLogic(int randomNumber, int guess) {
-   // todo: implement GameLogic
+   if ( guess > randomNumber) {
```
Comment (chinloong93) [inline GuessingGame.cpp:15]: Remove space before guess

File: GuessingGame.cpp [hunk 2]
```diff
@@ -12,7 +12,14 @@ int GenerateRandomNumber() {
 // If guess is smaller than randomNumber, print to console "Too low."
 // If guess us equal to randomNumber, print to console "That's right"
 void GameLogic(int randomNumber, int guess) {
-   // todo: implement GameLogic
+   if ( guess > randomNumber) { 
+      std::cout << "Too high.\n" << std::endl; 
+   } else if ( guess < randomNumber ) {
+      std::cout << "Too low.\n" << std::endl;
```
Comment (chinloong93) [inline GuessingGame.cpp:18]: Change too low to too little

**Review by chinloong93 — APPROVED** [review]

File: GuessingGame.cpp (hunk 1 above)
Comment (bktansuki) [reply GuessingGame.cpp:15]: ok

File: GuessingGame.cpp (hunk 2 above)
Comment (bktansuki) [reply GuessingGame.cpp:18]: ok

**Pull request merged**